tests/test.env

```diff
@@ -60,3 +60,5 @@ DMARC_CHECK_ENABLED=true
 PROTON_CLIENT_ID=to_fill
 PROTON_CLIENT_SECRET=to_fill
 PROTON_BASE_URL=https://localhost/api
+
+POSTMASTER=postmaster@simplelogin.co
```

Comment:
Nits: should we use a dummy value for this param?

Reply:
Changed